internal/datasource/common/pluginconfiguration/plugin_configuration_schema.go

```diff
@@ -54,6 +63,12 @@ func ToDataSourceSchema() schema.SingleNestedAttribute {
 													Optional:    false,
 													Computed:    true,
 												},
+												"encrypted_value": schema.StringAttribute{
+													Description: "For encrypted or hashed fields, this attribute contains the encrypted representation of the field's value, if a value is defined. If you do not want to update the stored value, this attribute should be passed back unchanged.",
```

Comment:
You can remove the "If you do not want to update..." sentence here and below, since there's no way to update anything with a data source